fe-nc-news-react/src/components/FullArticle.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { fetchArticlebyId as fetchArticlesId } from '../api';
+import CircularProgress from "@mui/material/CircularProgress";
+import Box from "@mui/material/Box";
+
+export default function FullArticle({ isLoading, setIsLoading}) {
+    const [article, setArticle] = useState({})
+
+    const { article_id } = useParams()
+
+    useEffect(() => { 
+        setIsLoading(true);
+        fetchArticlesId(article_id).then((res) => {
```

Comment:
rather than sending back the res here we should send back only the article from our fetch - as the function name suggests this is what we are returning from it